eng/ManualVersions.props

```diff
@@ -0,0 +1,20 @@
+<Project>
+
+<!--
+  The following properties (unlike those in version.props) are maintained manually. So
+  when there is a merge conflict, the highest version should generally be chosen, rather
+  than keeping what's in the branch, which is the usual strategy with version.props merge
+  conflicts.
+
+  Basically: In this file, choose the highest version when resolving merge conflicts.
+ -->
+  <PropertyGroup>
+    <MicrosoftWindowsSDKNETRef10_0_17763PackageVersion>10.0.17763.31</MicrosoftWindowsSDKNETRef10_0_17763PackageVersion>
+    <MicrosoftWindowsSDKNETRef10_0_18362PackageVersion>10.0.18362.31</MicrosoftWindowsSDKNETRef10_0_18362PackageVersion>
+    <MicrosoftWindowsSDKNETRef10_0_19041PackageVersion>10.0.19041.31</MicrosoftWindowsSDKNETRef10_0_19041PackageVersion>
+    <MicrosoftWindowsSDKNETRef10_0_20348PackageVersion>10.0.20348.31</MicrosoftWindowsSDKNETRef10_0_20348PackageVersion>
+    <MicrosoftWindowsSDKNETRef10_0_22000PackageVersion>10.0.22000.31</MicrosoftWindowsSDKNETRef10_0_22000PackageVersion>
+    <MicrosoftWindowsSDKNETRef10_0_22621PackageVersion>10.0.22621.31</MicrosoftWindowsSDKNETRef10_0_22621PackageVersion>
+  </PropertyGroup>
```

Comment:
Why havent we migrated Build.props yet? Is that needed from the installer repo? I'm not sure where the installer build is added within the pipeline, I was unable to find it.

Reply:
The 2 sections in `Build.props` from Installer have to do with:
- The first is VMR related and the *new* plan is to give the VMR team some time to work with that effort on this PR
- The second is related to the build process of the Installer repo. We don't use that same process in this repo. Instead, the `finalizer-build.csproj` is directly included in the SLN here and it is not in Installer repo.

This file is included at the very bottom of `eng/Versions.props`. I talked with the team that manages this file and they wanted the contents to remain in this file. They update the versions in this file every 6 months or so. The contact is Manodasan Wignarajah.

Reply:
Ah I see, thank you for clarifying. That explains a lot as to why its no longer there! :)